### ISSUE TITLE
Is Equalify just another WCAG testing service?

### ISSUE THREAD

**Issue by bbertucc** [bbertucc/equalify#4]
Nope! We enforce WCAG standards through deep integrations with your site's content management systems. For instance, we can email site owners with WCAG errors. Testing services without Equalify's deep integration cannot do that. 

We plan to offer lots more WebOps integrations. You'll be able to manage every aspect of every website in your network.
